src/typescript/11-keyof-typeof/keyof-typeof.md

```diff
@@ -67,10 +70,13 @@ const tRex = {
   weightInKilograms: 7_000,
 };
 
-let stegosaurus: typeof tRex; // {name: string; type: string; weightInKilograms: number;}
+let stegosaurus: typeof tRex; 
 
-type Dinosaur = typeof tRex; // {name: string; type: string; weightInKilograms: number;}
+type Dinosaur = typeof tRex; 
 ```
+@highlight 7, 9
+
+Thanks to the use of `typeof`, the variable `stegosaurus` and the type `Dinosaur` both now has the type: `{name: string; type: string; weightInKilograms: number;}`
```

Comment:
`has` -> `have`?

Also, maybe put the type in a separate code block so it’s easier to read?